src/main/java/com/github/black0nion/blackonionbot/systems/antiswear/AntiSwearSystem.java

```diff
@@ -50,7 +50,7 @@ public static boolean check(final BlackGuild guild, final BlackMember author, fi
 		try {
 			if (messageContent.equalsIgnoreCase("")) return false;
 			// check for whitelist
-			final List<String> whitelist = guild.getAntiSwearWhitelist();
+			final List<String> whitelist = guild.getAntiSwearWhitelist().getValue();
```

Comment:
can you not have it were getValued is called automatically instead having to always called e.g getAntiSwearWhitelist() calls getValue.
This applies to all the other getValues.

Reply:
Already thought about that, but I'm using Lombok rn which doesn't support that, but I'll have a look at it

Reply:
I would advise against using Lombok. It throws a compile error when using its ```@getters and @setter ``` .